lib/airbrake/passenger.rb

```diff
@@ -0,0 +1,62 @@
+require 'set'
+require 'pathname'
+
+module Airbrake
+  # Integration for Phusion Passenger
+  module Passenger
+    IntegrationException = Class.new(::Exception)
```

Comment:
It's entirely possible there's a better way to do it, but it's raised on lines 11 and 40 and caught in the '#integrate!` method.